test/integration.test.js

```diff
@@ -258,19 +258,29 @@ describe('with access key', function () {
             4000000000,  // fundingAmount
         );
         await nearjs.waitForTransactionResult(addAccessKeyResponse);
+       
```

Comment:
let's avoid code duplication in tests. E.g.:

```
        // Adding access key
        const keyForAccessKey = KeyPair.fromRandomSeed();
        const addAccessKeyResponse = await account.addAccessKey(
            newAccountId,
            keyForAccessKey.getPublicKey(),
            contractId,
            '',  // methodName
            '',  // fundingOwner
	            4000000000,  // fundingAmount
	        );
```

this can easily be in `beforeEach` for whole set of tests. This will also help differentiate what is setup for a test case and what is actually tested.

```
        // test that we can't make calls with deleted key
	        const contract = await nearjs.loadContract(contractId, {
	            viewMethods: ['getValue'],
	            changeMethods: ['setValue'],
	            sender: newAccountId,
	        });
```

this can be a helper function

Reply:
this is a function call. I don't want to make a helper function for a single function call.

Reply:
first snippet is 10 line of code for repeated over at least 3 tests, can be replaced by zero additional lines for each test. There is hardcoded stuff like funding amount which very likely needs to be updated for all tests at once vs individually. 

and second snippet is  5 lines of code used also about 3 times. Stuff like `viewMethods: ['getValue'],
	            changeMethods: ['setValue'],` also is boilerplate which likely will need to get replaced for all places at once and not individually, as it's specific to .wasm file deployed (and we don't have different files for different testS).